packages/web-pixels-extension/src/schemas/pixel-events.ts

```diff
@@ -457,7 +457,7 @@ export const pixelEvents = {
       },
     },
     EventType: {
-      enum: ['standard', 'dom', 'custom'],
+      enum: ['standard', 'dom', 'custom', 'error'],
```

Comment:
@kiftio these look like your changes. Are you okay if I make these schema changes public?

Reply:
Hey @SimonGaudert we'd ideally keep it internal. I'm not sure how easy that'd be though.

@leandrooriente do we think exposing just the enum value would be OK?

Reply:
I can manually remove the enum changes from this PR and my subsequent dev docs PR.

Reply:
That would be great, thanks for that 🙏

Reply:
By the way, if it's not much effort to include, I noticed this typo that could do with fixing - https://github.com/Shopify/web-pixels-manager/pull/930

I'll finish off the PR otherwise

Reply:
Created a standalone [PR](https://github.com/Shopify/web-pixels-manager/pull/931) to take care of this.